src/content/docs/logs/get-started/logging-best-practices.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Logging best practices guide
+tags:
+ - New Relic solutions
+ - Best practices guides
+ - Logs
+ - Logging
+translate:
+ - jp
+metaDescription: 'Best practices for using New Relic logs'
+redirects:
+---
+
+import parsingExample from 'images/logs-parsing-example.png'
+import image from 'images/image.png'
+
+Welcome to the New Relic logging best practices guide. Here you'll find recommendations for:
+
+- [Forwarding logs](#forwarding-logs)
+- [Data partitions](#partitions)
+- [Parsing logs](#parsing-logs)
+- [Drop filter rules](#drop-rules)
+- [New Relic log sizing](#sizing)
```

Comment:
Is there a reason for the "New Relic" preface, or can this title just be "Log sizing"?

Reply:
Yeah it can just be Log sizing